README.md

```diff
@@ -1,2 +1,27 @@
-# .github
-Readme information for TheCookieMC
+## Welcome to TheCookieMC!
+
+![CookieMC banner](https://raw.githubusercontent.com/TheCookieMC/BugReports/main/cookiemc.png?token=GHSAT0AAAAAABZTNNIHRBVS66ROEBB6OHVSY3O5CUA)
```

Comment:
Not a publicly visible link, please provide a link on a public image hosting website.

Reply:
okay on it :)